doc/articles/working-with-accessibility.md

```diff
@@ -35,6 +35,15 @@ FeatureConfiguration.AutomationPeer.UseSimpleAccessibility = true;
 #endif
 ```
 
+##Disabling accessibility text scaling
+
+You have the option to disable accessibility text scaling of iOS and Android devices but Apple [recommends to keep text sizes dynamic](https://developer.apple.com/videos/play/wwdc2017/245) 
+
+Here's how to disable it
+```csharp
+// App's constructor (App.xaml.cs) 
+Uno.UI.FeatureConfiguration.Font.IgnoreTextScaleFactor= true; 
+```
 We highly recommend using this mode, as iOS still won't let you focus nested accessible elements even if you don't (see known issues).
```

Comment:
This last phrase is now in the wrong section no? "We Highly recommend using this more...", it should be above ##Disabling accessibility text scaling

Reply:
should be resolved now.